src/main/java/org/galatea/starter/service/IexService.java

```diff
@@ -45,5 +46,14 @@
     }
   }
 
-
+  /**
+   * Get the historical prices for each Symbol and date range that is passed in.
+   *
+   * @param symbol the list of symbols to get a last traded price for.
+   * @param dateRange the range of dates to search for historical prices.
+   * @return a list of historical prices objects for each Symbol that is passed in.
+   */
+  public List<IexHistoricalPrices> getHistoricalPrices(final String symbol, final String dateRange, final String date) {
+    return iexClient.getHistoricalPrices(symbol,dateRange,date);
```

Comment:
Better style to have spacing between method parameters

Reply:
Missed that, will fix.